JS/Vardges/FitnesApplication/src/MainPages/FitnessAppContainers/FoodPage/General.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import FoodForm from './FoodForm';
+import FoodList from './FoodList';
+import useLocalStorage from './uselocaleStorage'
+
+
+function Food({dark}) {
+   const [items, setTodos] = useLocalStorage(' ', [])
```

Comment:
Why you send those arguments to 'useLocalStorage' function, could you please explain ? I am quite not sure that you understood what you did

Reply:
I save the user's food list  in items object so that the list is not deleted after reloading page.

Reply:
@VardgesKarapetyan jan and what is the key for that object?